consensus/spos/bls/subroundBlock.go

```diff
@@ -104,6 +109,18 @@ func (sr *subroundBlock) doBlockJob() bool {
 }
 
 func (sr *subroundBlock) sendBlock(body data.BodyHandler, header data.HeaderHandler) bool {
+	if sr.couldBeSentTogether(body, header) {
```

Comment:
No, they are not marshalized the second time. Now we keep body and headerHandler in clear in the ConsensusMessage struct. The consensus message will be marshalized before broadcast but this it will be done anyway. We need to extend the functionality of the BlockSizeComputation to calculate the size for a given Header too. For now we can evaluate only mini blocks and txs there. So, I need to be accurate here in the calculation of the message size as from this point I have received a lot of messages from p2p with buffer to large